model/event/subscribers.go

```diff
@@ -17,6 +17,15 @@ const (
 	EvergreenWebhookSubscriberType  = "evergreen-webhook"
 	EmailSubscriberType             = "email"
 	SlackSubscriberType             = "slack"
+	NoSubscriberType                = "none"
```

Comment:
I don't like the idea of NoSubscriberType because it's a non-zero type that is technically invalid (i.e. it's not a subscriber"). If my memory is correct, the subscriber model will also fail to unmarshal something with nil data.

Reply:
Fixed. I initially thought I would need a way to differentiate between "no change" and "no notifications" in the route but that's not needed